src/scheduling/schedule_config.py

```diff
@@ -52,10 +49,20 @@ def read_weekdays(incoming_schedule_config):
 
 def read_schedule_config(incoming_schedule_config):
     repeatable = read_repeatable_flag(incoming_schedule_config)
-    start_datetime = _read_start_datetime(incoming_schedule_config)
+    start_datetime = _read_datetime(incoming_schedule_config, 'start_datetime')
```

Comment:
Please add validation for start_datetime is not None. It was a part of `_read_datetime` before, but you removed the validation

Reply:
My bad I'll re add, tho I belive currently it's not possible to even pass None as if you leave the fields empty it grabs the default values from page load as well date is not required.

![image](https://github.com/bugy/script-server/assets/38784343/ebafe608-40ce-4dca-b212-6f702592b6a8)